src/validation.cpp

```diff
@@ -342,11 +342,15 @@ static bool IsCurrentForFeeEstimation(CChainState& active_chainstate) EXCLUSIVE_
  * Passing fAddToMempool=false will skip trying to add the transactions back,
  * and instead just erase from the mempool as needed.
  */
-
-static void UpdateMempoolForReorg(CChainState& active_chainstate, CTxMemPool& mempool, DisconnectedBlockTransactions& disconnectpool, bool fAddToMempool) EXCLUSIVE_LOCKS_REQUIRED(cs_main, mempool.cs)
+void CChainState::UpdateMempoolForReorg(
+    CChainState& active_chainstate,
```

Comment:
In commit "refactor: move UpdateMempoolForReorg into CChainState" (7cdcd08d665570167b06a93e003369f7fa833cec)

`active_chainstate` is always `*this` right? Can the parameter be dropped?

Reply:
Good catch, thanks. Always weary to assume active_chainstate == *this, but in this case (as you probably realized) only the active ever has a mempool, so it's a fine substitution.